jetty-unixsocket/pom.xml

```diff
@@ -21,6 +21,15 @@
           <onlyAnalyze>org.eclipse.jetty.unixsocket.*</onlyAnalyze>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <systemPropertyVariables>
+            <org.eclipse.jetty.unixsocket.LEVEL>DEBUG</org.eclipse.jetty.unixsocket.LEVEL>
```

Comment:
It's better to do this in a `src/test/resources/jetty-logging.properties` file.  There are plenty of examples in other modules.